pkgs/top-level/aliases.nix

```diff
@@ -1900,6 +1900,7 @@ mapAliases {
   pio = throw "pio has been removed due to lack of upstream maintenance"; # Added 2025-01-25
   pipewire_0_2 = throw "pipewire_0_2 has been removed as it is outdated and no longer used"; # Added 2024-07-28
   pipewire-media-session = throw "pipewire-media-session is no longer maintained and has been removed. Please use Wireplumber instead.";
+  platformioPackages = throw "platformioPackages was removed and its packages where moved to toplevel"; # Added 2025-09-04
```

Comment:
Should we construct an attrset of aliases instead? 😅

Reply:
Could do

Reply:
Ah, already swapped it over.